addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1518,6 +1518,11 @@ body.editor_enable.editor_has_snippets {
             we-list, we-list > div, we-list we-select {
                 margin-top: 0;
             }
+            &, we-list {
+                > div {
+                    display: block;
+                }
+            }
```

Comment:
Ok this solves the problem... but I don't really understand why putting display: block on this DIV works. Can you explain it to me ? I would have thought that the fix here would be to put `max-width: 100%` on `we-list` and `we-select`

Reply:
> Ok this solves the problem... but I don't really understand why putting display: block on this DIV works. Can you explain it to me ? I would have thought that the fix here would be to put `max-width: 100%` on `we-list` and `we-select`

You are right, the solution with `max-width: 100%` is better. I made the changes.